Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -145,12 +145,12 @@
   - type: UseDelay
     delay: 1.0
 
-# All using the same sprite until Bright gives me sprites the rest.
+# Swept <3s Bright
```

Comment:
Pretty cute but not sure if that's a good enough argument to merge otherwise useless comments